### ISSUE TITLE
🐛 Bug Report: Backblaze stores 2 files for each upload

### ISSUE THREAD

**Issue by rossanmol** [appwrite/appwrite#3694]
### 👟 Reproduction steps

1. Setup backblaze vars in .env
2. Upload a file in Appwrite Storage
3. Open backblaze bucket
4. Notice the file upload

### 👍 Expected behavior

1 file is uploaded to backblaze

### 👎 Actual Behavior

2 files are uploaded to backblaze, 1 original file and a weird file with same name.

### 🎲 Appwrite version

Version 0.15.x

### 💻 Operating system

MacOS

### 🧱 Your Environment

![image](https://user-images.githubusercontent.com/18351988/184963662-d04c2f96-a43e-4d9c-ba95-6be307fd67cf.png)

Screenshot for the double files

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by JoshiJoshiJoshi** [comment]
I can confirm this bug on my end. It looks like Appwrite is uploading the file twice to the backblaze server. 
One is encrypted and the other one isn't. 
My backblaze server is located in EU Central and I'm using the default bucket settings:
Encryption: disabled
Files: Private

![](https://cdn.discordapp.com/attachments/564161955963731980/1010956820908617798/unknown.png)
![](https://cdn.discordapp.com/attachments/564161955963731980/1010956821319651408/unknown.png)

**Comment by JoshiJoshiJoshi** [comment]
I had the suspicion that it was also downloading all the files twice. So I tested it by downloading the files through the admin console a few times and it seems to confirm my suspicion. On my end I only received one file but I guess the backblaze adapter is downloading 2 files in the background.

![](https://cdn.discordapp.com/attachments/564161955963731980/1010987025228374077/unknown.png)
![](https://cdn.discordapp.com/attachments/564161955963731980/1010987025622630410/unknown.png)

**Comment by stnguyen90** [comment]
@rossanmol and @JoshiJoshiJoshi, what are your encryption and anti virus settings on the bucket? What are your anti virus settings from the .env file?

**Comment by JoshiJoshiJoshi** [comment]
@stnguyen90 both settings are enabled in the appwrite console, but the anti virus is actually disabled in the .env file.
<img width="257" alt="Screenshot 2022-08-23 at 23 20 43@2x" src="https://user-images.githubusercontent.com/49132782/186269299-b92edbbe-6856-4917-afe2-e9fa2b325db8.png">
<img width="267" alt="Screenshot 2022-08-23 at 23 19 28@2x" src="https://user-images.githubusercontent.com/49132782/186269311-99858e38-cdb7-4465-89bd-ed01acc4e841.png">

Disabling them doesn't make a difference. In fact, if you disable "Encryption" it still uploads two files and one is still encrypted.

**Comment by stnguyen90** [comment]
@JoshiJoshiJoshi that seems quite odd that it's still encrypted. 👀 Are you sure you clicked on save? Does viewing the bucket JSON show encryption is false?

**Comment by JoshiJoshiJoshi** [comment]
@stnguyen90 I double checked it before posting it because I found it weird as well 😉
<img width="769" alt="Screenshot 2022-08-24 at 08 25 09@2x" src="https://user-images.githubusercontent.com/49132782/186346009-946b7431-9d88-4288-b62f-420ca33ce5e3.png">

Restarted the docker container and issue still persist. 
**Tested file types:** jpeg, png, pdf, zip, tar.gz

**Comment by stnguyen90** [comment]
@JoshiJoshiJoshi, actually, I have a feeling it's gzipped rather than encrypted:

https://github.com/appwrite/appwrite/blob/e658457b1b8617c390bdd357a2010b1c80658a0b/app/controllers/api/storage.php#L510-L511

**Comment by JoshiJoshiJoshi** [comment]
@stnguyen90 doesn't look like I can open it if it's just gzipped

**Comment by stnguyen90** [comment]
@JoshiJoshiJoshi, did decompress it fail?

**Comment by JoshiJoshiJoshi** [comment]
@stnguyen90 indeed, none of the files can be decompressed. Besides that, all the files which can't be openend are actually bigger than the actual file, so I kinda doubt it's just gzipped.

**Comment by eldadfux** [comment]
Stale issue, closing down.
